docs/index.md

```diff
@@ -208,13 +111,15 @@ The Operator IAM roles will be created per cluster with the [rosa-sts](https://r
 ## Variables for Red Hat Cloud Services Terraform provider
```

Comment:
@JohnStrunk where can I find descriptions of what these variables do? All of these are generated from whatever the developer supplied + some review from docs, but we may be missing some items.

CC: @nirarg for next week

Reply:
The parameters are defined here: https://github.com/terraform-redhat/terraform-provider-rhcs/blob/main/provider/provider.go#L57
The documentation for what they do is pretty minimal though.

It looks like they are largely a copy of the parameters used for rosa cli and ocm cli. Perhaps there is info that can be taken from those docs?

```console
$ rosa login --help
Log in to your Red Hat account, saving the credentials to the configuration file.
The supported mechanism is by using a token, which can be obtained at: https://console.redhat.com/openshift/token/rosa

The application looks for the token in the following order, stopping when it finds it:
	1. Command-line flags
	2. Environment variable (ROSA_TOKEN)
	3. Environment variable (OCM_TOKEN)
	4. Configuration file
	5. Command-line prompt

Usage:
  rosa login [flags]

Examples:
  # Login to the OpenShift API with an existing token generated from https://console.redhat.com/openshift/token/rosa
  rosa login --token=$OFFLINE_ACCESS_TOKEN

Flags:
      --client-id string       OpenID client identifier. The default value is 'cloud-services'.
      --client-secret string   OpenID client secret.
      --govcloud               Uses the FedRAMP High OpenShift Cluster Manager API for creating clusters in AWS GovCloud regions
  -h, --help                   help for login
      --insecure               Enables insecure communication with the server. This disables verification of TLS certificates and host names.
      --region string          Use a specific AWS region, overriding the AWS_REGION environment variable.
      --scope strings          OpenID scope. If this option is used it will completely replace the default scopes. Can be repeated multiple times to specify multiple scopes. (default [openid])
  -t, --token string           Access or refresh token generated from https://console.redhat.com/openshift/token/rosa.
      --token-url string       OpenID token URL. The default value is 'https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token'.

$ ocm login --help
Log in, saving the credentials to the configuration file.
The recommend way is using '--token', which you can obtain at: https://console.redhat.com/openshift/token

Usage:
  ocm login [flags]

Flags:
      --client-id string       OpenID client identifier. The default value is 'cloud-services'.
      --client-secret string   OpenID client secret.
  -h, --help                   help for login
      --insecure               Enables insecure communication with the server. This disables verification of TLS certificates and host names.
      --password string        User password.
      --persistent             By default the tool doesn't persistently store the user name and password, so when the refresh token expires the user will have to log in again. If this option is provided then the user name and password will be stored persistently, in clear text, which is potentially unsafe.
      --scope strings          OpenID scope. If this option is used it will replace completely the default scopes. Can be repeated multiple times to specify multiple scopes. (default [openid])
      --token string           Access or refresh token.
      --token-url string       OpenID token URL. The default value is 'https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token'.
      --url string             URL of the API gateway. The value can be the complete URL or an alias. The valid aliases are 'production', 'staging', 'integration' and their shorthands. (default "https://api.openshift.com")
      --user string            User name.
```